experiment/pod.yaml

```diff
@@ -0,0 +1,46 @@
+# Sample file for running e2e jobs in k8s
+# Steps:
+#   1. Prepare service-account.json and ssh-keys (priv, pub)
+#   2. Prepare a cluster (`gcloud container clusters create CLUSTER_NAME`)
+#   3. `kubectl create secret generic service-account --from-file=service-account.json=/path/to/service-account.json`
+#      `kubectl create secret generic ssh-key-secret --from-file=ssh-private=/path/to/rsa --from-file=ssh-public=/path/to/rsa.pub`
+#   4. Make sure your job sets `--mode=local` in config.json
+#   5. Replace JOB_NAME and --timeout in the yaml
+#   6. kubectl create -f pod.yaml
+#
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: k8s-e2e
+spec:
+  containers:
+  - name: k8s-e2e-test
+    image: gcr.io/k8s-testimages/kubekins-e2e-prow:0.3
```

Comment:
Different name that matches the common part of the job name (`kubernetes-e2e` not `kubekins-e2e`).

Do we need to support multiple versions for the e2e tests? We don't right now.